packages/server/lib/automation/commands/key_press.ts

```diff
@@ -136,17 +145,27 @@ export async function bidiKeyPress ({ key }: KeyPressParams, client: Client, aut
   }
 
   try {
+    const chars = NamedKeys.includes(inKey) ? [key] : [...key]
+
     await client.inputPerformActions({
       context: autContext,
       actions: [{
         type: 'key',
-        id: `${autContext}-${key}-${idSuffix || Date.now()}`,
-        actions: [
-          { type: 'keyDown', value },
-          { type: 'keyUp', value },
-        ],
+        id: `${autContext}-${inKey}-${idSuffix || Date.now()}`,
```

Comment:
The logical OR encourages the use of `''` as a suffix. Also, will `Date.now()` collide if multiple calls are made in the same millisecond?


```suggestion
        id: `${autContext}-${inKey}-${idSuffix ?? crypto.randomUUID()}`,
```

Reply:
Logical OR results in there *always* being a suffix, even if `''` is passed in - an empty string is falsey. A nullish coalescing operator will result in `context-key-`, which is not quite what we want here.

TBH, the BiDi spec is very vague on what this `id` needs to be, and given the nature of how Cypress' server works, it's very unlikely to collide. I'm okay with switching it over to a random uuid. However, `Date.now()` is what was there previously, so that kind of refactor (including test updates) should probably be covered in a separate PR.